ssl/record/ssl3_record.c

```diff
@@ -977,6 +977,8 @@ int tls1_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int sending)
     unsigned char padval;
     int imac_size;
     const EVP_CIPHER *enc;
+    int tlstree_enc = (sending ? (s->mac_flags & SSL_MAC_FLAG_WRITE_MAC_TLSTREE)
+                      : (s->mac_flags & SSL_MAC_FLAG_READ_MAC_TLSTREE));
```

Comment:
I think the outer parens are unnecessary,
and I would like the align the : below the ?

Reply:
Fixed. Thanks!